pkgs/servers/typedb/default.nix

```diff
@@ -57,23 +64,28 @@ stdenv.mkDerivation rec {
   src = srcFolder;
 
   dontBuild = true;
+  #phases = [unpackPhase installPhase installCheckPhase];
+  unpackPhase = ''
```

Comment:
@roberth I am still having problems with the unpack phase even if I use dont build:
```
-build --option build-use-sandbox true -A typedb
these derivations will be built:
  /nix/store/srgpxriy2k5ysd4f509k0wrcbhvl2cgd-typedb-2.1.1.drv
building '/nix/store/srgpxriy2k5ysd4f509k0wrcbhvl2cgd-typedb-2.1.1.drv'...
unpacking sources
here
env-vars
here2
tar: typedb-all-linux-2.1.1.tar.gz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
builder for '/nix/store/srgpxriy2k5ysd4f509k0wrcbhvl2cgd-typedb-2.1.1.drv' failed with exit code 2
error: build of '/nix/store/srgpxriy2k5ysd4f509k0wrcbhvl2cgd-typedb-2.1.1.drv' failed
```
the archive is not in scope yet again;
is this still connected to the phases or is it something different?

Reply:
You can use the default `unpackPhase`. No need to override it.
`installPhase` will have these files in the working directory

```
drwxr-xr-x 4 nixbld nixbld  4096 Jan  1  2000 .
drwxr-xr-x 3 nixbld nixbld  4096 Jan  1  2000 ..
-rwxr-xr-x 1 nixbld nixbld 34520 Jan  1  2000 LICENSE
drwxr-xr-x 4 nixbld nixbld  4096 Jan  1  2000 console
drwxr-xr-x 5 nixbld nixbld  4096 Jun 10 08:03 server
-rwxr-xr-x 1 nixbld nixbld  4618 Jan  1  2000 typedb
```